src/main/java/com/salesforce/mirus/config/SourceConfig.java

```diff
@@ -104,4 +107,19 @@ public String getName() {
     }
     return transformations;
   }
+
+  private static String parseTopicsRegex(String topicsRegex) {
+    if (!topicsRegex.startsWith("[") || !topicsRegex.endsWith("]")) {
```

Comment:
Would it be easier to parse this with Jackson? We can specify the input as a JSON array.

Reply:
Yeah, I also thought about it. Just not sure whether the extra `""` around each regex worth it. regex is hard to read, the additional symbol could make it worse.
Let's forget about how the code implemented. If we only look at the regex config, is JSON array easier to read than the current config `[abc_cde, fgh_ijk]`?

Reply:
Personally, I think it is easier to read a JSON array. But that's a personal preference :)

Reply:
I suppose another option is to just use a Comma Separated List, then there is no `[` , `]` or `"`. This is how other configuration for Kafka is done.

Reply:
Whatever we do it must be backward compatible. This is a public open-source project so we can't afford to break any users' existing configurations. Note that `[` and `,` are valid regex characters and could already appear in existing config.

Reply:
I removed the need for `[ ]` as Jeremy suggested this would be align with what Kafka is doing.
Just saw Paul's comment, `,` is a valid regex char but it's not a [valid Kafka topic char](https://github.com/apache/kafka/blob/0.10.2/core/src/main/scala/kafka/common/Topic.scala#L29) to use, so I think it should be safe.

Reply:
What if an existing user has a topic regex using the repetition operator, something like this: `topic_{1,3}name`, which matches on between 1 and 3 underscores. Does Kafka have a standard way to escape the comma? Would it be safer to use a different config parameter name?

Reply:
I'm okay with a breaking change as this is going to be bundled in the major upgrade to 2.6.